openslides/users/static/templates/users/group-list.html

```diff
@@ -15,48 +18,56 @@ <h1 translate>Groups</h1>
     <thead>
       <tr>
-        <th ng-click="sortby='id';reverse=!reverse">
-          <translate>ID</translate>
-          <i class="fa" ng-show="sortby == 'id'"
-              ng-class="reverse ? 'fa-caret-down' : 'fa-caret-up'"></i>
-        <th ng-click="sortby='name';reverse=!reverse">
-          <translate>Name</translate>
-          <i class="fa" ng-show="sortby == 'name'"
-              ng-class="reverse ? 'fa-caret-down' : 'fa-caret-up'"></i>
-        <th os-perms="users.can_manage core.can_manage_projector" class="mini_width" translate>Actions</th>
-    <tbody>
-      <tr ng-repeat="group in groups | filter: filter.search | orderBy:sortby:reverse">
-        <td>{{ group.id }}
-        <td><a ui-sref="users.group.detail({id: group.id})">{{ group.name | translate }}</a>
-        <td os-perms="users.can_manage" class="nobr">
-          <!-- edit -->
-          <a ui-sref="users.group.detail.update({id: group.id})" os-perms="users.can_manage"
-              class="btn btn-default btn-sm"
-              title="{{ 'Edit' | translate }}">
-            <i class="fa fa-pencil"></i>
-          </a>
-          <!-- delete -->
-          <a os-perms="users.can_manage" class="btn btn-danger btn-sm"
+        <!-- Fixed width, so the table doesn't change width when expanding an app.
+             It's a work-around for Chrome(ium) because visibility: collapse doesn't render correct. -->
+        <th style="width: 300px;">
+          <h4 translate>Permissions</h4>
+        <th ng-repeat="group in groups" ng-mouseover="group.hover=true" ng-mouseleave="group.hover=false">
+          {{ group.name | translate }}
+          <i class="fa fa-info-circle" ng-if="group.id==1"
+            uib-tooltip="{{ 'All users without any group or permission gain permissions from this group.' | translate }}"></i>
+          <div os-perms="users.can_manage" class="hoverActions"
+            ng-class="{'hiddenDiv': !group.hover}">
+            <!--edit name-->
+            <a href="" ng-bootbox-prompt="{{ 'Please enter a new name for' | translate }} <b> {{group.name | translate }}</b>:"
+              ng-bootbox-prompt-action="changeName(result, group)">
+                <i class="fa fa-pencil"></i>
+            </a>&nbsp;
+            <!--delete-->
+            <a href="" class="text-danger"
               ng-bootbox-confirm="{{ 'Are you sure you want to delete this entry?' | translate }}<br>
-              <b>{{ group.name }}</b>"
-              ng-bootbox-confirm-action="delete(group)"
-              title="{{ 'Delete' | translate }}">
-            <i class="fa fa-trash-o"></i>
-          </a>
+              <b>{{ group.name | translate }}</b>"
+              ng-bootbox-confirm-action="delete(group)" translate>
+                <i class="fa fa-trash fa-lg"></i>
+            </a>
+          </div>
+      <tbody ng-repeat="app in apps" os-perms="users.can_manage">
+          <tr class="pointer" ng-click="app.app_visible=!app.app_visible">
```

Comment:
Suggestion: Just fill only the app name row in grey (and the permission rows in white). This could help to delimit the apps.